testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -335,3 +335,8 @@ def test_missing_endif(self):
         with pytest.raises(IOError):
             with self.parser(ITP_no_endif) as p:
                 top = p.parse(include_dir=GMX_DIR)
+
+class Testrelativepath:
+
+    def test_relpath(self):
+        u = mda.Universe('../data/gromacs_ala10.itp', format='ITP')
```

Comment:
Where is this itp file being generated?

Reply:
Okay, I get it. I guess I need to load this file first..!

Reply:
@aditya-kamath Python treats files with relative paths, as relative to whichever directory you've called the test from. Users can call `pytest` from anywhere in their system, which is why we make all our paths absolute in the `datafiles.py` file and import them when we want to use our test files. It might be easier for you to write a temporary ITP file using [pytest's tools](https://docs.pytest.org/en/stable/tmpdir.html) that only exists for the scope of this function or module, and try to read that.

Reply:
@lilyminium , This really helps. Now, I understand how to write this test! :)

Reply:
I've now created a temporary folder where an itp file is written and have accessed it with the parser. Although, I am still not completely sure if this asserts that itp parser can access relative locations.?